CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ cmake_minimum_required(VERSION 3.5.1)
 
 project(falco)
 
-option(USE_BUNDLED_DEPS "Bundle hard to find dependencies into the Falco binary" OFF)
+option(USE_BUNDLED_DEPS "Bundle hard to find dependencies into the Falco binary" ON)
```

Comment:
In reality, this was somehow `ON` because :sweat_smile: :
- we actually used the value coming from libs https://github.com/falcosecurity/libs/blob/master/CMakeLists.txt#L56 
- we forced everything from libs to ON here: 
https://github.com/falcosecurity/falco/blob/master/cmake/modules/falcosecurity-libs.cmake#L84
- the option was not used in Falco cmake modules https://github.com/falcosecurity/falco/tree/master/cmake/modules
- and all Falco modules https://github.com/falcosecurity/falco/tree/master/cmake/modules we already bundled by default

This change basically makes everything more consistent and aligns with what was already happening.

Indeed, this change won't affect the default behavior, and the workflow of client code should be unchanged.